### PR TITLE
Remove p-footer and associated classes. Update MAAS exmaple

### LINKED ISSUE #1500: Replace footer component with existing components
Currently the footer links are the same as text. They should have a color, underline or both.

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Remove p-footer as it can be replicated with other components.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/templates/maas-layout/
- Check the the footer using generic components looks good.

## Details

Fixes #1500

### CODE REVIEW

**Comment by kwm14** [comment]
Design LGTM 👍

File: scss/_utilities_sticky-footer.scss [hunk 1]
```diff
@@ -1,17 +0,0 @@
-@mixin vf-u-sticky-footer {
```
Comment (Caleb-Ellis) [inline scss/_utilities_sticky-footer.scss]: Did we decide we want to remove a sticky footer utility as well?

Comment (deadlight) [reply scss/_utilities_sticky-footer.scss]: The sticky footer util only related to the footer component. This reverts sticky footers to being a site-specific customisation.

Comment (Caleb-Ellis) [reply scss/_utilities_sticky-footer.scss]: Oh ok, in that case it all looks good EXCEPT there's still a mention of `$sticky-footer` in the spacing settings file

Comment (deadlight) [reply scss/_utilities_sticky-footer.scss]: Ok, removed

**Comment by 0atman** [comment]
rad^sick

**Pull request merged**